src/codeu/chat/client/simplegui/Conversation.java

```diff
@@ -0,0 +1,108 @@
+package codeu.chat.client.simplegui;
+
+/**
+ * @author  Troy Jones
+ * @date    5/21/17
+ * @brief   Contains all GUI necessary to execute the frame to 
+ *          enable chat.
+ */
+
+import java.awt.BorderLayout;
+import java.awt.Button;
+import java.awt.Container;
+import java.awt.FlowLayout;
+import java.awt.GridBagConstraints;
+import java.awt.GridBagLayout;
+import java.awt.event.MouseAdapter;
+import java.awt.event.MouseEvent;
+
+import javax.swing.BoxLayout;
+import javax.swing.JButton;
+import javax.swing.JFrame;
+import javax.swing.JLabel;
+import javax.swing.JPanel;
+import javax.swing.JTextArea;
+import javax.swing.JTextField;
+
+@SuppressWarnings("serial")
+
+public class Conversation extends JFrame {
+
+    boolean notClickedYet = true;
```

Comment:
are you using this anywhere else except within this class? if not, please mark this as private

Also, why not initialize the value in the constructor?
Furthermore, can you add a comment for what this is supposed to be?  It's not clear what "notClickedYet" is supposed to do

And... while you're at it, perhaps rename to isFirstClick?